### DISCUSSION TITLE
Apache NetBeans 17 release candidates

### DISCUSSION THREAD

**Discussion by neilcsmith-net** [apache/netbeans#5324] [General]
Release candidates for the next release of Apache NetBeans will be available weekly until release. Your help in testing, reporting and verifying issues would be highly appreciated by the whole Apache NetBeans community.

## Binary / source zip

Binary and source zips are available from -

https://nightlies.apache.org/netbeans/candidate/netbeans/

## Installers

Unsigned Windows, Linux DEB and Linux RPM installers are available from -

https://nightlies.apache.org/netbeans/candidate/installers/

NB. We are working on making other installers available for release candidates from the same location.

## VSCode plugin

The VSCode plugin is available from - 

https://nightlies.apache.org/netbeans/candidate/vsix/

## IMPORTANT

These release candidates are pre-release software, and not official releases of Apache NetBeans.

**Comment by mbien** [comment]
release notes are now also available for RCs as pre-releases, for example 17-rc1:
https://github.com/apache/netbeans/releases/tag/17-rc1

**Comment by neilcsmith-net** [comment]
17-rc2 is now available.  I've also added the deb and rpm builds to the installer section, and updated the text in the main post.

**Comment by lkishalmi** [comment]
Unfortunately we have some Snapcraft authentication issues on the Apache build servers, so no Snap produced this time.

**Reply by lkishalmi** [reply]
I've opted to upload the Apache built binary to Snapcraft myself yesterday so Apache NetBeans 17-rc2 is available on the edge channel at:
https://snapcraft.io/netbeans

**Comment by neilcsmith-net** [comment]
Apache NetBeans 17-rc3 is now available and links have been updated. Changes between 17-rc2 and 17-rc3 at https://github.com/apache/netbeans/releases/tag/17-rc3
